webview-ui/src/features/deep-research/GetStarted.tsx

```diff
@@ -0,0 +1,278 @@
+													<div className="text-muted-foreground">
+														Firecrawl turns websites in LLM-ready data.
+													</div>
+													<Button variant="outline" size="sm" asChild>
+														<a href="https://www.firecrawl.com/">Get API Key</a>
+													</Button>
+												</div>
+											</div>
+										)}
+									/>
+								</CollapsibleContent>
+							</Collapsible>
+						</Card>
+					) : (
+						<Card className="flex-row items-center">
+							<CircleBackslashIcon className="text-destructive shrink-0" />
+							<div className="text-muted-foreground">
+								Deep research requires usage of an OpenAI model. Please create an OpenAI or OpenRouter
+								profile to get started.
+							</div>
+						</Card>
+					)}
+
+					<Card title="Deep Research Parameters">
+						<Controller
+							name="breadth"
+							control={control}
+							render={({ field: { value, onChange } }) => (
+								<div className="flex flex-row items-center gap-3">
+									<div className="w-24 whitespace-nowrap shrink-0 text-right">
+										Breadth <span className="text-muted-foreground">({value})</span>
+									</div>
+									<Slider
+										min={1}
+										max={10}
+										step={1}
+										value={[value]}
+										onValueChange={(values) => onChange(values[0])}
+									/>
+								</div>
+							)}
+						/>
+						<Controller
+							name="depth"
+							control={control}
+							render={({ field: { value, onChange } }) => (
+								<div className="flex flex-row items-center gap-3">
+									<div className="w-24 whitespace-nowrap shrink-0 text-right">
+										Depth <span className="text-muted-foreground">({value})</span>
+									</div>
+									<Slider
+										min={0}
+										max={9}
+										step={1}
+										value={[value]}
+										onValueChange={(values) => onChange(values[0])}
+									/>
+								</div>
+							)}
+						/>
+						<Controller
+							name="concurrency"
+							control={control}
+							render={({ field: { value, onChange } }) => (
+								<div className="flex flex-row items-center gap-3">
+									<div className="w-24 whitespace-nowrap shrink-0 text-right">
+										Concurrency <span className="text-muted-foreground">({value})</span>
+									</div>
+									<Slider
+										min={1}
+										max={5}
+										step={1}
+										value={[value]}
+										onValueChange={(values) => onChange(values[0])}
+									/>
+								</div>
+							)}
+						/>
+						{provider && <Models className="w-full" />}
+						<Controller
+							name="query"
+							control={control}
+							render={({ field }) => (
+								<AutosizeTextarea
+									{...field}
+									placeholder="What would you like me to research?"
+									minHeight={75}
+									maxHeight={200}
+									className="p-3"
+								/>
+							)}
+						/>
+					</Card>
+
+					<Button type="submit">
+						<RocketIcon />
```

Comment:
I know this is inconsistent in other ways, but it felt a little jarring to see a rocket that was not the codicon rocket

Reply:
Fixed.